$MirOS: ports/comms/linphone/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- configure.in.orig	Wed Mar 31 20:22:33 2004
+++ configure.in	Wed Oct  6 00:04:51 2004
@@ -14,7 +14,7 @@ AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
 AC_HEADER_STDC
-AC_PROG_INTLTOOL
+dnl AC_PROG_INTLTOOL
 dnl Initialize libtool
 AC_PROG_LIBTOOL
 AM_PROG_LIBTOOL
@@ -94,9 +94,9 @@ dnl os-specific problems not handled by 
 PTHREAD_LDFLAGS=
 GSMLIB_CFLAGS="-w -O2"
 case "$host_os" in
-	*freebsd*)
+	*freebsd* | *openbsd* | *mirbsd*)
 		PTHREAD_LDFLAGS="-pthread"
-		AC_DEFINE(__FreeBSD__,1,[Defined if we are compiling for FreeBSD])
+dnl		AC_DEFINE(__FreeBSD__,1,[Defined if we are compiling for FreeBSD])
 		;;
 	*linux*)
 		AC_DEFINE(__LINUX__,1,[Defined if we are compiling for linux])
@@ -212,6 +212,7 @@ if test "$alsa" = "true"; then
 		]
 	)
 fi
+AC_CHECK_LIB(ossaudio,_oss_ioctl,[LIBS="$LIBS -lossaudio"])
 
 dnl check if we have the getifaddrs() sytem call
 AC_CHECK_FUNCS(getifaddrs)
